packages/@okta/vuepress-site/guides/sign-into-native-ui/configure-the-callback/iOS/configurescheme.md

```diff
@@ -0,0 +1,3 @@
+To do this, open `Info.plist `in your application bundle and set a **URL Scheme** to the scheme of the login redirect URI (for example, {yourOktaScheme}).
+
+For example, if your **Login Redirect URI** is `com.okta.example:/callback`, the **URL Scheme** is `com.okta.example`.
```

Comment:
@IldarAbdullin-okta Is this still required for a custom sign-in UI project, even though there is no browser redirect involved?

Reply:
It is required for later exchanging session token to access token which is a logical completion of sign-in flow. Do you think it is confusing?